.github/workflows/cd.yml

```diff
@@ -0,0 +1,105 @@
+name: CD
+
+on:
+  release:
+    types: ["published"]
+
+jobs:
+  build:
+    name: Build
+    runs-on: macos-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Setup Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: "20"
+          cache: "yarn"
+      - name: Install dependencies
+        run: yarn install --frozen-lockfile
+      - name: Build
+        run: yarn release
+      - name: Upload chrome artifact
+        uses: actions/upload-artifact@v2
+        with:
+          path: bundle/chrome
+          name: chrome
+      - name: Upload firefox artifact
+        uses: actions/upload-artifact@v2
+        with:
+          path: bundle/firefox
+          name: firefox
+      - name: Upload safari artifact
+        uses: actions/upload-artifact@v2
+        with:
+          path: bundle/safari
+          name: safari
+  
+  chrome:
+    name: Deploy to Chrome
+    runs-on: ubuntu-latest
+    needs: build
+    env:
+      EXTENSION_ID: ${{ secrets.CHROME_EXTENSION_ID }}
+      CLIENT_ID: ${{ secrets.CHROME_CLIENT_ID }}
+      CLIENT_SECRET: ${{ secrets.CHROME_CLIENT_SECRET }}
+      REFRESH_TOKEN: ${{ secrets.CHROME_REFRESH_TOKEN }}
+    steps:
+      - name: Download artifact
+        uses: actions/download-artifact@v2
+        with:
+          name: chrome
+      - name: Upload to Chrome Web Store
+        run: npx chrome-webstore-upload-cli upload --auto-publish
+
+#   edge:
+#     name: Deploy to Edge
+#     runs-on: ubuntu-latest
+#     needs: build
+#     steps:
+#       - name: Download artifact
+#         uses: actions/download-artifact@v2
+#         with:
+#           name: chrome
+
+  firefox:
+    name: Deploy to Firefox
+    runs-on: ubuntu-latest
+    needs: build
+    env:
+      API_KEY: ${{ secrets.FIREFOX_API_KEY }}
+      API_SECRET: ${{ secrets.FIREFOX_API_SECRET }}
+    steps:
+      - name: Download artifact
+        uses: actions/download-artifact@v2
+        with:
+          name: firefox
+      - name: Sign add-on
+        run: npx web-ext sign --api-key $API_KEY --api-secret $API_SECRET
+      - name: Upload signed add-on
+        uses: actions/upload-artifact@v2
+        with:
+          path: web-ext-artifacts
+          name: firefox-signed
+
```

Comment:
Should we trigger a `nitsig-website` action to update itself, here?

Reply:
We should find a way to update the website automatically, I think triggering an action there is possible like you said. I'll investigate.